src/dashboard/src/middleware/common.py

```diff
@@ -27,18 +28,29 @@
 from django.utils.deprecation import MiddlewareMixin
 from shibboleth.middleware import ShibbolethRemoteUserMiddleware
 
+
 import elasticsearch
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Please use this form:

    logger = logging.getLogger("archivematica.dashboard")

Otherwise it'll be given a name we don't want to use.